css/index.css

```diff
@@ -344,18 +354,60 @@ footer nav a {
   text-decoration: none;
 }
 
-.middle-content .boxes .box {
+.top-content1 {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  margin-bottom: 20px;
+}
+
+.top-content1 .text-container {
+  width: 800px;
+  padding: 0 1%;
+  padding-bottom: 20px;
+  border-bottom: 1px dashed black;
+}
+
+.middle-content1 {
+  display: flex;
+  flex-direction: column;
```

Comment:
No need for `flex-direction: column` on `.middle-content1`. To get the grid layout, use `flex-wrap: wrap` and `justify-content: space-around`